Uplift/Models/Gym.swift

```diff
@@ -253,6 +253,29 @@ struct FacilityDetail {
             subfacilities = detailData.subFacilityNames.compactMap({ $0 })
         }
     }
+
+    func getEquipmentCategories() -> [EquipmentCategory] {
+       var equipmentDict = [String: [Equipment]]()
+        equipment.forEach { equipmentItem in
+            if equipmentDict[equipmentItem.equipmentType] != nil {
+                equipmentDict[equipmentItem.equipmentType]?.append(equipmentItem)
+            } else {
+                equipmentDict[equipmentItem.equipmentType] = [equipmentItem]
+            }
+        }
+
+        var equipmentCategories = equipmentDict.compactMap { tuple in
+            return EquipmentCategory(categoryName: tuple.key, equipment: tuple.value.sorted {
```

Comment:
Nit: get rid of `return` inside closures with only one expression. Can also shorten the expression by doing something like:
```
return equipmentDict.compactMap { ... }.sort { ... }
```